MIGRATION-TO-SCK.md

```diff
@@ -0,0 +1,322 @@
+# Migration from Splunk Connect for Kubernetes
```

Comment:
Change to "Splunk OpenTelemetry Connector for Kubernetes". This is the official product name.